front/components/data_source/DeleteStaticDataSourceDialog.tsx

```diff
@@ -51,21 +59,44 @@ export function DeleteStaticDataSourceDialog({
   }, [isUsageLoading, isUsageError, usage, name]);
 
   return (
-    <Dialog
-      alertDialog
-      isOpen={isOpen}
-      title={`Removing ${name}`}
-      onValidate={onDelete}
-      isSaving={isLoading || isUsageLoading}
-      onCancel={onClose}
-      validateVariant="warning"
+    <NewDialog
+      open={isOpen}
+      onOpenChange={(open) => {
+        if (!open) {
+          onClose();
+        }
+      }}
     >
-      <div>
-        {message}
-        <br />
-        <br />
-        <b>Are you sure you want to remove ?</b>
-      </div>
-    </Dialog>
+      <NewDialogContent>
+        <NewDialogHeader>
+          <NewDialogTitle>Confirm deletion</NewDialogTitle>
+        </NewDialogHeader>
+        {isLoading ? (
+          <div className="flex justify-center py-8">
+            <Spinner variant="dark" size="md" />
+          </div>
+        ) : (
+          <>
+            <NewDialogContainer>
+              {message}
```

Comment:
NIT: We removing the 2 br ?

Reply:
Yeah, the default `NewDialog` layout already has some spacing